pkg/util/database.go

```diff
@@ -0,0 +1,148 @@
+package database
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+)
+
+// Json file as a database to store key-value pairs
+type JsonData struct {
+	m       sync.RWMutex
+	dataMap map[string]string
+}
+
+// Database interface
+type Database interface {
+	Init() error
+	Read() (map[string]string, error)
+	Write(map[string]string) error
+	Contains(key string) bool
+}
+
+// DB file name
+var JSON_DB = "kvstore.json"
```

Comment:
Golang has some style convention regarding names. You should use camel case everywhere. For exported names its upper camel case, for non exported names its lower camel case.